ops/build_and_package.sh

```diff
@@ -42,19 +42,34 @@ go build -ldflags "-X github.com/CMSgov/bcda-app/bcda/constants.Version=$VERSION
 echo "Packaging bcda binary into RPM..."
 fpm -v $VERSION -s dir -t rpm -n bcda bcda=/usr/local/bin/bcda swaggerui=/etc/sv/api _site=/etc/sv/api
 cd ../bcdaworker
-go clean 
+go clean
 echo "Building bcdaworker..."
 go build
 echo "Packaging bcdaworker binary into RPM..."
 fpm -v $VERSION -s dir -t rpm -n bcdaworker bcdaworker=/usr/local/bin/bcdaworker
+cd ../ssas
+go clean
+echo "Building ssas..."
+go build -o ssas ./service/main
+echo "Packaging ssas binary into RPM..."
+fpm -v $VERSION -s dir -t rpm -n ssas ssas=/usr/local/bin/ssas
 
 #Sign RPMs
-WORKER_RPM="bcdaworker-*.rpm"
 echo "Importing GPG Key files"
 /usr/bin/gpg --batch --import $GPG_PUB_KEY_FILE
 /usr/bin/gpg --batch --import $GPG_SEC_KEY_FILE
 /usr/bin/rpm --import $GPG_PUB_KEY_FILE
 
+SSAS_RPM="ssas-*.rpm"
```

Comment:
I was thrown by the change in order here. That is, when building the packages, the order is (bcda, bcdaworker, ssas), but when signing the packages, the order is (ssas, bcdaworker, bcda). Is the change in order significant? Can we make the order consistent so no one asks this question again?

Reply:
The order was reversed before adding ssas; I suppose to avoid changing directories an extra time. I was just following what was already in place, but can update this if it makes sense to everyone.